internal/pkg/cli/deploy/errors.go

```diff
@@ -46,7 +46,10 @@ type errEnvHasPublicServicesWithRedirect struct {
 }
 
 func (e *errEnvHasPublicServicesWithRedirect) Error() string {
-	return fmt.Sprintf("%v services redirect HTTP to HTTPS", len(e.services))
+	return fmt.Sprintf("%v %s HTTP to HTTPS",
```

Comment:
Do we need to include the service names  in the error msg? Also btw is this included in any unit test?

Reply:
I don't think so - service name(s) are printed in `RecommendActions()`. there is a test [here](https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/cli/deploy/env_test.go#L561-L617).